src/components/image/image.js

```diff
@@ -62,7 +65,14 @@ export default class Image extends PureComponent {
           [`is-${s}`]: s,
         })}
       >
-        <img onError={this.onError} src={this.state.src} alt={alt} />
+        <img
+          className={classnames(className, {
```

Comment:
I'm not sure about adding the same classmates to the figure and the img tag. Maybe just adding the  is-rounded will be enough. Wdyt?

Reply:
@couds yes you're right; will fix asap.